cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -251,17 +244,18 @@ To delete a specific version of an application, submit an HTTP DELETE that inclu
      - Description
    * - ``namespace-id``
      - Namespace ID
-   * - ``application-name``
+   * - ``app-id``
      - Name of the application to be deleted
    * - ``version-id``
      - Version of the application to be deleted
 
-**Note:** The ``application-name`` in this URL is the name of the application
-as configured by the application Specification,
-and not necessarily the same as the name of the JAR file that was used to deploy the application.
-This does not delete the streams and datasets associated with the application
-because they belong to the namespace, not the application.
-Also, this does not delete the artifact used to create the application.
+**Note:** The ``application-name`` in this URL is the name of the application as
```

Comment:
There are a number of places on this page that use `-name` instead of `-id`; I have now changed just about all (only two are left , lines #117 and #139) where it seemed to me to make sense to leave them as-is.